catalog/core/catalog-core-api-impl/pom.xml

```diff
@@ -46,6 +48,42 @@
             <groupId>ddf.platform.util</groupId>
             <artifactId>platform-util</artifactId>
         </dependency>
+        <dependency>
+            <groupId>ddf.security.core</groupId>
+            <artifactId>security-core-api</artifactId>
+        </dependency>
+
+        <!--Groovy dependencies-->
+        <dependency>
```

Comment:
Should these groovy/spock dependencies be moved to dependency management in the reactor pom? Or at least should we create some version properties?